bin/cli

```diff
@@ -49,6 +49,21 @@ async function updatePackageJson (packageJsonPath) {
   await fs.writeJson(packageJsonPath, newPackageJson, packageJsonFormat)
 }
 
+async function initialiseGitRepo () {
+  const presenceCheckOutput = []
+  const notPresentText = 'not present'
+  await exec(`which git || echo "${notPresentText}"`, {}, data => presenceCheckOutput.push(data))
```

Comment:
I wonder if it's possible to write a test for this 🤔

Reply:
It's difficult with git being present on those machines, I can write something that mocks `exec` but it adds complexity without giving me a lot of confidence.